### DISCUSSION TITLE
baseball player totals please?

### DISCUSSION THREAD

**Discussion by SportspunterDotCom** [cloudbet/docs#33] [Ideas]
Hi Guys,
Can you please add baseball player totals to the feed (eg pitcher strikeouts, batter hits, total bases etc etc etc... whatever player totals there are)?

thanks a lot!

**Comment by kgravenreuth** [comment]
@SportspunterDotCom 

Thanks for bringing over your request here. For the Cloudbet Feed API, we currently offer the markets listed in our Markets list here: https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json

We add additional markets from time to time, such as the Tennis Game Handicap and Total Games markets added today. 

This is also mentioned in the README for this repo for your reference: https://github.com/Cloudbet/docs#markets-sports-and-categories-list

At present, we are not planning to add the Baseball Player Totals markets to the list of markets on our Feed API. We will update this thread if this changes in the future. Please refer to the available Baseball markets in the Markets list mentioned above.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
